source/session.ts

```diff
@@ -307,7 +306,8 @@ export class Session {
       return out;
     }
 
-    if (data && data._isAMomentObject) {
+    const date = convertToISOString(data);
```

Comment:
Should we try to convert all strings and numbers, or should we separate checking for date-like objects and encoding to iso string?

Reply:
Not sure what you mean by this comment. currently we support converting all date objects, including moment and dayjs that have `toISOString` method - as well as converting ISO 8601 compatible strings to UTC. Do you want to split strings and number cases from object cases?

Reply:
Basically, the question was if we should do:

```
if (isDateLike(data)) {
    data = convertToISOString(data)
}
```


Rather than:
```
    date = convertToISOString(data)
    if (date) {
        data = date;
    }

```

I am not convinced that we should. I just thought it looked odd to try converting all types (except arrays and pojo:s already handled).

Reply:
I don't mind either way, so let me know if you want me to change it

Reply:
Let's keep it